### DISCUSSION TITLE
v1.0.0

### DISCUSSION THREAD

**Discussion by InvisibleRain** [InvisibleRain/Tetris-in-Excel#7] [General]
##### System requirements:
* Excel 2007 or later
* Microsoft Windows

##### How to play?
1. Download the .xlsm file
2. Open it in Excel, then click "Allow Editing" and "Allow Content"
3. Play!!!

<hr /><em>This discussion was created from the release <a href='https://github.com/InvisibleRain/Tetris-in-Excel/releases/tag/v1.0.0'>v1.0.0</a>.</em>

**Comment by drumsmasher** [comment]
Hi all. I'm using Mac 14.2 with Microsoft Excel 365. Having this error when hit 'start'
![image](https://github.com/InvisibleRain/Tetris-in-Excel/assets/86668666/25389661-8229-4df8-bea7-3abeea39b8a6)

The line says:
```
 If GetAsyncKeyState(Down) Then
        Call tetr.Move("down")
```

**Reply by InvisibleRain** [reply]
Unfortunately, VBA depends on Windows-only user32.dll for handling key presses as seen in this module:
https://github.com/InvisibleRain/Tetris-in-Excel/blob/master/src/Modules/Engine.vba

I'll update README shortly

**Reply by InvisibleRain** [reply]
Here, I finally ported the game to LibreOffice Calc. You can try it out on any platform (Including MacOS!)

In order to play, you need to allow macros first:
1. Open LibreOffice Calc
2. Press Alt+F12 to open settings
3. Go to "Security" -> "Macro Security"
4. Set the Security Level to Medium
5. Apply the settings, restart Calc and run [Tetris.ods](https://github.com/InvisibleRain/Tetris-in-Excel/files/14056891/Tetris.ods)

**Reply by drumsmasher** [reply]
Thanks! I'll have to try it in my linux laptop.
